### ISSUE TITLE
Checkbox filter does not work when using dynamic values from Ajax data

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sukanyapoolakanda/jquery-datatables-column-filter#155]
```
What steps will reproduce the problem?
1. Set up environment to use data from an ajax source.
2. Add checkbox option with values set to null or left out.
3. View datatable.

What is the expected output? What do you see instead?
Should see a button to click with dynamically created checkboxes.
A button is not displayed.

What version of the product are you using? On what operating system?
Latest version.

Please provide any additional information below.

If I set up the project to use sample data from the source checkbox.html file, 
it works. As soon as I change it to use an ajax source, it does not.

So basically, if the data already exists within the table itself, it works, but 
does not work when passing ajax data.

The button is not created and if you inspect the element, nothing is there. 
Looking at the console, no errors appear.


```

Original issue reported on code.google.com by `frew.rob...@gmail.com` on 10 Apr 2014 at 9:00
